src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security
+{
+    public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        protected CodeAccessPermission() { }
+        public void Assert() { }
+        public abstract System.Security.IPermission Copy();
+        public void Demand() { }
+        public void Deny() { }
```

Comment:
I concede the point.

---
In reply to: [70849940](https://github.com/dotnet/corefx/pull/10049#discussion_r70849940) [](ancestors = 70849940)